apps/api/package.json

```
@@ -78,7 +78,7 @@
     "@types/chai": "^4.2.11",
     "@types/express": "4.17.7",
     "@types/mocha": "^8.0.1",
-    "@types/node": "^14.6.0",
+    "@types/node": "16.11.7",
```

Did you encounter some errors without this change? I'm all in to upgrade the @types/node but I think it's out of the scope of this PR. What do you think?

---

I have no idea about this.

---

If that's possible let's revert this back until we decided to migrate this across the board

```suggestion
    "@types/node": "^14.6.0",
```